docs/API-standaarden.md

```diff
@@ -3,21 +3,37 @@ layout: landing-page
+Een Application Programming Interface (API) is een manier om twee of meer computerprogramma's met elkaar te laten communiceren. Het is een soort software-interface die een dienst aanbiedt aan andere stukjes software.
 
-Vanuit de VNG wordt op verschillende manieren gewerkt aan de ontwikkeling van gestandaardiseerde API's: 
+De API's werken volgens de [REST-architectuurstijl](https://en.wikipedia.org/wiki/Representational_state_transfer) en volgen de [REST-API Design Rules](https://www.forumstandaardisatie.nl/open-standaarden/rest-api-design-rules) standaard.
 
-* Via het programma HaalCentraal is gewerkt aan het ontwikkelen van de [HaalCentraal API's](https://vng-realisatie.github.io/Haal-Centraal/) zodat gemeenten beter data kunnen opvragen uit de basisregistraties. 
-* Het project Notificatieservices heeft een concept-standaard ontwikkeld voor het notificeren van plaatsgevonden gebeurtenissen, zoals het wijzigen van brongegevens, aan daarop geabonneerde afnemers. Deze [NL GOV for CloudEvents standaard](https://logius.nl/domeinen/gegevensuitwisseling/nl-gov-profile-cloudevents) is nu in beheer bij Logius. 
-* Voor ondersteuning van zaakgericht werken binnen gemeenten zijn [API-standaarden voor zaakgericht werken](https://vng.nl/projecten/zaakgericht-werken-api) ontwikkeld.
+## Hoe maakt VNG Realisatie API-standaarden
+
+Zie:
+
+- [API-Community](API-Community)
+- [API-beoordelingsaspecten](API-beoordelingsaspecten)
+- [API-kwaliteitscriteria](API-kwaliteitscriteria)
+- [Beheer API-specificatie]Beheer_API-specificatie
 
-## Overzicht vastgestelde API standaarden
 Op de website van VNG.nl staat een [overzicht van alle door het College van Dienstverleningszaken door het VNG-bestuur of de Algemene Ledenvergadering van de VNG vastgestelde API-standaarden](https://vng.nl/artikelen/overzicht-gemeentelijke-standaarden). Er staat uitgelegd welke statussen er zijn en hoe het proces tot [proces van standaardverklaring](https://vng.nl/brieven/proces-standaardverklaring) is vastgesteld door het College van Dienstverleningszaken op 12 oktober 2018.
+
+## Welke API-standaarden
+
+Vanuit de VNG wordt op verschillende manieren gewerkt aan de ontwikkeling van gestandaardiseerde API's:
+
+- Via het programma HaalCentraal is gewerkt aan het ontwikkelen van de [HaalCentraal API's](https://vng-realisatie.github.io/Haal-Centraal/) zodat gemeenten beter data kunnen opvragen uit de basisregistraties. 
+- Het project Notificatieservices heeft een concept-standaard ontwikkeld voor het notificeren van plaatsgevonden gebeurtenissen, zoals het wijzigen van brongegevens, aan daarop geabonneerde afnemers. Deze [NL GOV for CloudEvents standaard](https://logius.nl/domeinen/gegevensuitwisseling/nl-gov-profile-cloudevents) is nu in beheer bij Logius. 
+- Voor ondersteuning van zaakgericht werken binnen gemeenten zijn [API-standaarden voor zaakgericht werken](https://vng.nl/projecten/zaakgericht-werken-api) ontwikkeld.
+
+## Compliancy
+
+API testvoorziening
```

Comment:
Ik neem aan dat je hier nog iets meer mee wil dan alleen deze 2 woordjes.

Reply:
De [API-testvoorziening](https://github.com/VNG-Realisatie/api-test-platform) ondersteunt ontwikkelaars bij het controleren en valideren van API's. Door het uitvoeren van tests kunnen potentiële problemen vroegtijdig worden geïdentificeerd en opgelost. De API-testvoorziening wordt vervolgens gebruikt om aan te tonen dat de gerealiseerde API compliant is met de API-standaard.

Reply:
W.m.b. prima tekst maar ik ben eerlijk gezegd niet zo ingewijd in de API-testvoorziening. Indien je daar behoefte aan hebt kun je het beste Henri vragen of deze tekst voldoet.